sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -19186,7 +19186,9 @@ public partial class BlobServiceProperties
         /// <summary>
         /// The set of CORS rules.
         /// </summary>
+#pragma warning disable CA2227 // Collection properties should be readonly
```

Comment:
@pakrym, This file is auto-generated.  We will have to re-apply this change each time we generate, unless the .NET generator is updated to generate this annotation.

Reply:
Good point, I have a generator update branch at https://github.com/Azure/azure-sdk-for-net/pull/17181

But it seems the generated code is not up-to-date in other cases as well.